### PR TITLE
update of design details

### PR DESCRIPTION
**Pull request by stanvins**

### CODE REVIEW

File: src/client/components/filter/TagsSuggester.component.js [hunk 1]
```diff
@@ -278,7 +278,9 @@ class TagsSuggester extends React.Component {
     const inputProps = {
       id: 'Searchbar__inputfield',
       type: 'search',
-      placeholder: 'Søg på titel, forfatter, stemning...',
+      placeholder: this.props.selectedFilters.length
+        ? ''
```
Comment (kfudbc) [inline src/client/components/filter/TagsSuggester.component.js]: Behøver placeholder at være tom streng?

File: src/client/components/top/TopBar.component.js [hunk 2]
```diff
@@ -276,7 +276,7 @@ export class TopBar extends React.Component {
             href="/find"
             className="Topbar__navigation__btn d-i-block d-md-none"
           >
-            <Icon name="search" onClick={() => this.toggleSearchBar('open')} />
+            <Icon name="search" onClick={() => this.toggleSearchBar('open')}/>
```
Comment (kfudbc) [inline src/client/components/top/TopBar.component.js]: Er prettier sat korrekt op?

File: src/client/components/filter/TagsSuggester.component.js (hunk 1 above)
Comment (stanvins) [reply src/client/components/filter/TagsSuggester.component.js]: yes. when there is at least one tag, the placeholder should be hidden.

File: src/client/components/top/TopBar.component.js (hunk 2 above)
Comment (stanvins) [reply src/client/components/top/TopBar.component.js]: good question. I'll look into it.

**Pull request merged**